tests/unittests/tests-phydat/tests-phydat.c

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (C) 2018 Eistec AB
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+#include "embUnit.h"
+#include "tests-phydat.h"
+
+#include "phydat.h"
+
+#define ENABLE_DEBUG (0)
+#include "debug.h"
+
+static void test_phydat_fit(void)
+{
+    int val0 = 100000;
```

Comment:
Might be interesting to put a comment explaining what this test is supposed to do.

Reply:
addressed in #8592